apps/web/src/pages/Profile/ExperienceAndSkillsPage/components/AddExperienceDialog.tsx

```diff
@@ -0,0 +1,257 @@
+        id: "KmqF8d",
+        description: "Description for award section",
+      }),
+      buttonText: intl.formatMessage({
+        defaultMessage: "Add award",
+        id: "CSg8cH",
+        description: "Button text to add an award to the profile",
+      }),
+      buttonPath: `${paths.createAward(applicantId)}${applicationParam}`,
+    },
+  ];
+
+  const Footer = React.useMemo(
+    () => (
+      <Dialog.Close>
+        <IconButton color="purple" icon={ArrowLeftIcon}>
+          {intl.formatMessage({
+            defaultMessage: "Cancel and go back",
+            id: "tiF/jI",
+            description: "Close dialog button",
+          })}
+        </IconButton>
+      </Dialog.Close>
+    ),
+    [intl],
+  );
+
+  return (
+    <Dialog.Root defaultOpen={defaultOpen}>
+      <Dialog.Trigger>
+        <IconButton color="blue" icon={PlusIcon}>
+          {intl.formatMessage({
+            defaultMessage: "Add a new experience",
+            id: "bOAF9o",
+            description:
+              "Button to open modal to add a new experience to the profile",
+          })}
+        </IconButton>
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header
+          color="black"
+          subtitle={intl.formatMessage({
+            defaultMessage:
+              "Get started by selecting the type of experience you would like to add.",
+            id: "h5MtHc",
+            description: "Subtitle for the Add Experience dialog",
+          })}
+        >
+          {intl.formatMessage({
+            defaultMessage: "Add a new experience to your resume",
+            id: "yI5DFC",
+            description: "Heading for the Add Experience dialog",
+          })}
+        </Dialog.Header>
+        <div data-h2-margin="base(x1, 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Before you start describing your experience, first we would like to know what category it falls into. There are five options to choose from, you can add as many experiences to each category as you want.",
+            id: "MhgtB6",
+            description: "Instructions for the Add Experience dialog",
+          })}
+        </div>
+        <div
+          data-h2-margin="base(0, x0.5)"
+          data-h2-flex-grid="base(stretch, x2, x1)"
+        >
+          {experienceSections.map((section) => (
+            <div
+              key={section.title}
+              data-h2-margin-bottom="base(x2)"
+              data-h2-flex-item="base(1of1) p-tablet(1of2)"
+            >
+              <div
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+                data-h2-justify-content="base(space-between)"
+                data-h2-height="base(100%)"
+              >
+                <div>
+                  <div data-h2-margin="base(x0.5, 0)">
+                    <strong>
```

Comment:
It could be useful to convert this to a `<Heading level="h3" size="h6" />`

Reply:
@Jerryescandon with the smallest heading size the community heading wraps.  In the design the headers appear to have the same font size as their bodies.  Do you want to have a heading that wraps?  Or use the same font size as the body?  Or shorten it to just "Community experience"?
![image](https://user-images.githubusercontent.com/8978655/219145301-86ff39bc-5ffb-402b-aa17-756a54cf2d88.png)

Reply:
I think in this case we should visually change it to a **bold body**, however I would also make sure that we include wrapping but leave space for the icon
![image](https://user-images.githubusercontent.com/16977254/219421912-5af1adcc-2e2e-43ba-b13c-83b3929d90cc.png)